helix-view/src/editor.rs

```diff
@@ -162,10 +162,25 @@ pub struct Config {
     pub indent_guides: IndentGuidesConfig,
 }
 
-#[derive(Debug, Default, Clone, PartialEq, Serialize, Deserialize)]
-#[serde(rename_all = "kebab-case", deny_unknown_fields)]
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
+#[serde(default, rename_all = "kebab-case", deny_unknown_fields)]
 pub struct LspConfig {
+    /// Display LSP progress messages below statusline
     pub display_messages: bool,
+    /// Enable automatic pop up of signature help (parameter hints)
+    pub auto_signature_help: bool,
```

Comment:
Is there a way to manually trigger signature help? We can manually trigger completion with `ctrl-x`, maybe once we trigger that it automatically triggers signature help as well?

Reply:
I added a keybind earlier but [removed it](https://github.com/helix-editor/helix/pull/1755#issuecomment-1111196004) due to a conflict. Adding a keybind is as easy as binding it to `signature_help` command, and we definitely need a built-in insert mode keybind too. Any suggestions?

> We can manually trigger completion with `ctrl-x`, maybe once we trigger that it automatically triggers signature help as well?

I'd rather have a separate key that can toggle just the signature help popup since that allows exclusive control.